streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -223,7 +223,7 @@ private void validateStoreOpen() {
         try {
             return this.db.get(rawKey);
         } catch (final RocksDBException e) {
-            throw new ProcessorStateException("Error while getting value for key from store " + this.name, e);
+            throw new ProcessorStateException("Error while getting value for key %s from store " + this.name, e); // String format is happening in wrapping stores. So formatted message is thrown from wrapping stores.
```

Comment:
nit: put the comment in the line above to avoid too long lines :) (same below)

Reply:
Fixed :)